airflow/ui/src/app.tsx

```diff
@@ -17,19 +17,16 @@
  * under the License.
  */
 
-import { Box } from "@chakra-ui/react";
-import { DagsList } from "src/dagsList";
-import { Nav } from "src/nav";
+import { DagsList } from "src/pages/DagsList";
+import { Navigate, Route, Routes } from "react-router-dom";
+import { BaseLayout } from "./layouts/BaseLayout";
 
-export const App = () => {
-  return (
-    <div>
-      <Nav />
-      <Box p={3} ml={24}>
-        <DagsList />
-      </Box>
-    </div>
-  );
-};
-
-export const AppSimple = () => <div>Something</div>;
+// Note: When changing routes, make sure to update init_react_ui.py too
+export const App = () => (
+  <Routes>
+    <Route path="/" element={<BaseLayout />}>
+      <Route index element={<Navigate to="dags" />} />
+      <Route path="dags" element={<DagsList />} />
```

Comment:
WDYT about just making this the index instead?
```suggestion
      <Route index element={<DagsList />} />
```

Reply:
The Navigate is a placeholder to make sure we have the routing set up for when we add more pages.